bundle-workflow/jenkins_workflow/test/orchestrator/Jenkinsfile

```diff
@@ -0,0 +1,52 @@
+pipeline {
+    agent none
+    environment {
+        TEST_RUN_ID = "${BUILD_NUMBER}"
+    }
+    parameters {
+        string(name: 'opensearch_version')
+        string(name: 'build_id')
+        string(name: 'architecture')
+    }
+    stages {
+        stage('Execute Tests') {
+            steps {
+                parallel (
+                    IntegTests: {
+                        build job: 'integ-test',
+                        parameters: [
+                            string(name: 'opensearch_version', value: "${params.opensearch_version}"),
+                            string(name: 'build_id', value: "${params.build_id}"),
+                            string(name: 'architecture', value: "${params.architecture}"),
+                            string(name: 'test_run_id', value: "${TEST_RUN_ID}")
+                        ]
+                    },
+                    PerfTests: {
+                        build job: 'perf-test',
+                        parameters: [
+                            string(name: 'opensearch_version', value: "${params.opensearch_version}"),
+                            string(name: 'build_id', value: "${params.build_id}"),
+                            string(name: 'architecture', value: "${params.architecture}"),
+                            string(name: 'test_run_id', value: "${TEST_RUN_ID}")
+                        ]
+                    },
```

Comment:
- Do we want to get this in even though the job doesn't exist? I would prefer to hold off if this piece of code is un-used.
- Do we know if these params are good for perf test?

Reply:
* I have added a perf-test job in staging. The perf-test logic can be updated independent of the orchestrator DSL so it should be fine. Let me know if you have a strong preference otherwise.  
* These common params should be good. The configurations around which rally tests to run can be part of the `opensearch-build` repo as code, similar to the test_manifest.yml config. Keeping the DSL simple and uniform will help avoid the need for frequent jenkins deployments.

Reply:
Sure, if we have a strong sense that this is the code I am good with it. 
Feels like it is :)